src/Bicep.Core.UnitTests/Assertions/JTokenAssertionsExtensions.cs

```diff
@@ -69,12 +69,12 @@ public static AndConstraint<JTokenAssertions> HaveValueAtPath(this JTokenAsserti
             Execute.Assertion
                 .BecauseOf(because, becauseArgs)
                 .ForCondition(valueAtPath is not null)
-                .FailWith("Expected value at path {0} to be {1} but it was null", jtokenPath, expected.ToString());
+                .FailWith("Expected value at path {0} to be {1}{reason} but it was null. Original JSON: {2}", jtokenPath, expected.ToString(), instance.Subject?.ToString());
```

Comment:
Does `reason` always have a leading space?

Reply:
Yes (or is empty). I took this from the FluentAssertions codebase (see https://github.com/fluentassertions/fluentassertions/blob/c882c22c20622e1e7741945197a919356fc38f66/Src/FluentAssertions/Collections/CollectionAssertions.cs#L38 for example)